CMakeLists.txt

```diff
@@ -99,6 +99,10 @@ endif ()
 option(GOOGLE_CLOUD_CPP_GENERATE_DOXYGEN "Generate Doxygen documentation" OFF)
 mark_as_advanced(GOOGLE_CLOUD_CPP_GENERATE_DOXYGEN)
 
+# Compile the DocFX tool.
+option(GOOGLE_CLOUD_CPP_INTERNAL_DOCFX "Compile the docfx tool. Only google-cloud-cpp developers are expected to use this flag." OFF)
```

Comment:
Nit: I'm surprised `cmake-format` was happy with such a long line.

Reply:
Me too.  I think we are not formatting the top-level CMake file at all, I will send a separate PR to fix that.